src/go/rpk/pkg/cli/cmd/status.go

```diff
@@ -119,8 +119,9 @@ func executeStatus(
 		t.Append(row)
 	}
 
-	t.Render()
-
+	if !send {
```

Comment:
I don't see where send occurs. But it looks like `if !send` could be an early exit.

Reply:
@BenPope here: https://github.com/vectorizedio/redpanda/pull/329/files/23e4fecafe6f5701b51b330b61cd9f88bc614804#diff-d5153c54fde06d72d3bc5c53e8c3bf834ed8ea990c3a9f47418ec94cf56aaefcR105

Reply:
Ahh. I see.  Still, there seems to be a fair bit of work being done that doesn't need to be.